application/controllers/ConfigController.php

```diff
@@ -80,53 +70,69 @@ public function hasBeenSubmitted(): bool
         };
 
         $form = (new NotificationsConfigForm())
-            ->setKConfig($kconfig)
+            ->populate(['cluster_uuid' => $this->getRequest()->get('id')])
```

Comment:
Please rename `id` to `cluster_uuid` and retrieve the default value from the session. This ensures that any changes a user makes to the displayed cluster are automatically reflected when working with the config. If the session is empty, we still populate `null` so that the form defaults to the first database entry in the cluster table.